runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/LexerExec/CharSet.stg

```diff
@@ -1,21 +1,17 @@
 TestType() ::= "Lexer"
 
-Options ::= [
-	"Ignore": "Currently unable to generate the required CRLF input sequence."
-]
-
 Grammar ::= [
 	"L": {<grammar("L")>}
 ]
 
-Input() ::= "34<\u000D><\n> 34"
```

Comment:
right now those tests are marked as ignore precisely because cr is not working
I changed them so they can be ran as part of the build in every target

I had already done that in the original test generator

my suggestion is that we take the change, and once you have fixed the cr issue, you provide a dedicated test for cr

> Le 21 juin 2015 à 03:34, Sam Harwell <notifications@github.com> a écrit :
> 
> In runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/LexerExec/CharSet.stg <https://github.com/antlr/antlr4/pull/927#discussion_r32887262>:
> 
> >  Grammar ::= [
> >  	"L": {<grammar("L")>}
> >  ]
> >  
> > -Input() ::= "34<\u000D><\n> 34"
> Even if true, if carriage returns are working then there is no need to change this test. I prefer to address this change (and the other files with the same exclusion) separately.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/antlr/antlr4/pull/927/files#r32887262>.
>